go/src/koding/socialapi/presence/presence.go

```diff
@@ -0,0 +1,38 @@
+package presence
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"koding/socialapi"
+)
+
+type Client struct {
```

Comment:
Not without refactoring it. I need to have access to *http.Request. I can refactor it thought to allow for that. For now the requests are internally created, I order to request authentication I'd need to have access to requests' context:

https://github.com/koding/koding/pull/9863/files#diff-9da7fe549db0224bc2de4a603ce84d51R63